app/src/main/java/org/wikipedia/readinglist/LongPressMenu.kt

```diff
@@ -85,10 +87,16 @@ class LongPressMenu(private val anchorView: View, private val existsInAnyList: B
 
     private fun deleteOrShowDialog() {
         listsContainingPage?.let { list ->
-            RemoveFromReadingListsDialog(list).deleteOrShowDialog(anchorView.context) { _: List<ReadingList?>?, _: ReadingListPage? ->
+            RemoveFromReadingListsDialog(list).deleteOrShowDialog(anchorView.context) { readingLists, _ ->
                 entry?.let {
                     if (anchorView.isAttachedToWindow) {
-                        showMessage((anchorView.context as AppCompatActivity), anchorView.context.getString(R.string.reading_list_item_deleted, it.title.displayText))
+                        val readingListNames = mutableListOf<String>()
+                        readingLists.forEach { readingList ->
+                            readingListNames.add(readingList.title())
+                        }
```

Comment:
This could be simplified (see comment below).

Reply:
Is this resolved?

Reply:
yes @cooltey

Reply:
Sorry resolved now